### DISCUSSION TITLE
Is Snipe-IT vulnerable to the Log4J exploit?

### DISCUSSION THREAD

**Discussion by magicjmarker36** [snipe/snipe-it#10425] [Q&A] [answered]
Is Snipe-IT vulnerable to the Log4J exploit? I'm running it on Ubuntu 18.04 .

**Comment by snipe** [comment, marked as answer]
Snipe-IT is not, as Log4j is a java exploit, and we do not use any java. I cannot speak for what else might be running on your server tho.
